src/main/java/com/vesta/controller/impl/UserControllerImpl.java

```diff
@@ -22,64 +21,37 @@
 
     private final UserService userService;
 
+    private final UserCreateViewConverter userCreateViewConverter;
+
     private final UserViewConverter userViewConverter;
 
-    @ApiOperation(value = "Logarea pe pagina")
-    @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "Login was success"),
-            @ApiResponse(code = 404, message = "Username, password or email not found")
-    })
     public Map<String, String> login(AccountCredential accountCredential) {
         return userService.login(accountCredential);
     }
 
-    @ApiOperation(value = "Returneaza userul dupa ID")
-    @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "Get user by id has succeeded"),
-            @ApiResponse(code = 403, message = "Accessing the resource you were trying to reach is forbidden"),
-            @ApiResponse(code = 404, message = "User not found")
-    })
     public UserView getById(Long id) {
         return userViewConverter.convert(userService.getById(id));
     }
 
-    @ApiOperation(value = "Returneaza toti userii din baza de date")
-    @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "Get all users has succeeded"),
-            @ApiResponse(code = 404, message = "Users not found")
-    })
     public List<UserView> findAll() {
         return userService.findAll()
                 .stream()
                 .map(userViewConverter::convert)
                 .collect(Collectors.toList());
     }
-    @ApiOperation(value = "Creaza un nou user")
-    @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "User was created"),
-            @ApiResponse(code = 500, message = "Server error")
-    })
-    public void create(UserView userView) {
-        userService.create(userViewConverter.deconvert(userView));
+
+    public void create(UserCreateView userCreateView) {
+        userService.create(userCreateViewConverter.deconvert(userCreateView));
     }
-    @ApiOperation(value = "Modifica datele userului dupa ID")
-    @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "User was updated successfully"),
-            @ApiResponse(code = 404, message = "User not found")
-    })
-    public UserView update(Long id, UserView userView) {
-        return userViewConverter.convert(userService.update(id, userViewConverter.deconvert(userView)));
+
+    public UserCreateView update(Long id, UserCreateView userCreateView) {
```

Comment:
why you return UserCreateView at update? what you want to do with this response?

Reply:
because I have not fixed this thing yet, I understand everything, but I will not just fix the mistakes made by others, I have my tasks that do not make them on time, because I'm just fixing. ok I will fix, no problem but....sorry not now i want start Reset Password ticket